src/backends/caffe/caffelib.cc

```diff
@@ -3181,9 +3181,9 @@ namespace dd
                 int results_height = results[0]->height();
                 const int det_size = 7;
                 const float *outr = results[0]->cpu_data();
+                int k = 0;
```

Comment:
what does that "k" do? not sure to understand by reading the code, it seems like it prevents to go beyond `result_heights`, which is the total number of prediction in the batch? maybe add a comment or renaming the variable?

Reply:
exactly, so I guess no doc is needed because you had it w/o any help :D